routes/v1.js

```diff
@@ -496,9 +502,12 @@ api.declare({
     // otherwise the task would have been created with defineTask, and we don't
     // offer an idempotent operation in that case
     if (!_.isEqual(taskDef, def) || task.runs.length === 0) {
-      return res.status(409).json({
-        message:      "taskId: " + taskId + " already used by another task"
-      });
+      return res.reportError(
+        "RequestConflict",
+        "taskId {{taskId}} already used by another task.",
```

Comment:
This error message should probably state that it could be a faulty idempotency issues.